packages/esm-patient-chart-app/src/workspace/workspace-window.scss

```diff
@@ -133,12 +133,31 @@ html[dir='rtl'] {
       }
     }
   }
-  :global(.omrs-breakpoint-lt-desktop) {
-    .container {}
+  :global(.tab-container) {
```

Comment:
I'm a little nervous about using `:global()` in this context. Is it strictly necessary? This series of rules basically affects everything on the page. I'd prefer if we could find a way to either not do these globally or just do it in esm-styleguide in core.

Reply:
Yes, the idea is to apply this to every form that contains tabs. I will move it to esm-styleguide now, and I also think it can be done without using the global in esm-form-entry-app, but I couldn't run that module locally and see my changes. Do you have any advice on how to do it?

Reply:
Generally, you have to run both esm-patient-forms-app and esm-form-entry-app at the same time, so `yarn start --sources packages/esm-patient-forms-app --sources esm-form-entry-app`.